build/Build.Tests.cs

```diff
@@ -174,19 +172,54 @@ bool DoesSidHaveRightsToDirectory(string directory, WellKnownSidType sid, params
                     .Any(r => rights.Any(right => r.FileSystemRights.HasFlag(right)));
             }
 
-            FileSystemTasks.EnsureExistingDirectory(TestDirectory);
-            FileSystemTasks.EnsureCleanDirectory(TestDirectory);
+            Logging.InTest(nameof(TestWindowsInstallerPermissions), () =>
+            {
+                FileSystemTasks.EnsureExistingDirectory(TestDirectory);
+                FileSystemTasks.EnsureCleanDirectory(TestDirectory);
 
-            var installers = (ArtifactsDirectory / "msi").GlobFiles("*x64.msi");
+                var installers = (ArtifactsDirectory / "msi").GlobFiles("*x64.msi");
 
-            if (!installers.Any())
-            {
-                throw new Exception($"Expected to find at least one installer in the directory {ArtifactsDirectory}");
-            }
+                if (!installers.Any())
+                {
+                    throw new Exception($"Expected to find at least one installer in the directory {ArtifactsDirectory}");
+                }
             
-            foreach (var installer in installers)
-            {
-                TestInstallerPermissions(installer);
-            }
+                foreach (var installer in installers)
+                {
+                    TestInstallerPermissions(installer);
+                }
+            });
         });
+    
+    void RunTests()
+    {
+        Logger.Info($"Running test for Framework: {TestFramework} and Runtime: {TestRuntime}");
```

Comment:
Bit confused about where `TestFramework` and `TestRuntime` come from? Should they be params to this function?